Plugins/NativeCamera/Source/Apple/NativeCameraImpl.mm

```diff
@@ -26,18 +30,103 @@ @interface CameraTextureDelegate : NSObject <AVCaptureVideoDataOutputSampleBuffe
 }
 
 - (id)init:(std::shared_ptr<Babylon::Plugins::Camera::Impl::ImplData>)implData;
+- (id<MTLTexture>)getCameraTextureY;
+- (id<MTLTexture>)getCameraTextureCbCr;
 
 @end
 
+namespace {
+    static bool isPixelFormatSupported(uint32_t pixelFormat)
+    {
+        switch (pixelFormat) {
+            case kCVPixelFormatType_420YpCbCr8BiPlanarVideoRange:
+            case kCVPixelFormatType_420YpCbCr8BiPlanarFullRange:
+                return true;
+        }
```

Comment:
Should we add preferring full range over video range for the pixel format to the heuristic to select the best format?

Reply:
According to the docs here: https://developer.apple.com/documentation/avfoundation/capture_setup/choosing_a_capture_device?language=objc, `AVCaptureDeviceDiscoverySession` lists the best formats first and Apple puts video range before full range in the returned arrays. I'm not sure how Apple decides what's best but we should probably honor their decision instead of overriding it.

Reply:
Good to know it's already covered by the ordering in AVCaptureDeviceDiscoverySession